src/test/java/edu/harvard/iq/dataverse/api/GeoconnectIT.java

```diff
@@ -39,7 +41,7 @@ public void checkSingleMapLayerMetadata() {
                 .statusCode(OK.getStatusCode());
     }
 
-//    @Ignore
+    @Ignore
```

Comment:
Were these ignores added because the tests shouldn't be run normally?

Reply:
Yes, exactly. It's the same story with the Data Capture Module. The phoenix server doesn't have access to a running DCM or Geoconnect server, unfortunately.